inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html

```diff
@@ -34,7 +37,11 @@
       <div class="row">
         <div class="col-md-12 summary" id="detailed-header-top">
           <div class="col-md-2">
-            <img class="profile-image img-circle" src="{{ record['positions'][0]['email'] | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com' width="100%">
+            {% set email = '' %}
+            {% if record.get('positions') and (record['positions'])|length > 0 %}
```

Comment:
Generally, in jinja you strictly do not need to `.get` I believe. `record.positions | length > 0 ` may just work even if positions is not filled.